sphinx/io.py

```diff
@@ -155,7 +122,9 @@ def SphinxDummySourceClass(source, *args, **kwargs):
     return source
 
 
-class SphinxFileInput(FileInput):
+class SphinxBaseFileInput(FileInput):
```

Comment:
Now FileInput classes are separated for each file types. So this class becomes a super class of them.